py3status/modules/emerge_status.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+"""
+Emerge_status provides a short information on the current
+emerge status if there is an emerge process currently running
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in second (default 0)
+    format: Display format to user (default 'EMRG: {current} / {total}')
+    hide_if_zero: Don't show in bar if there is no emerge running (default False)
+
+Format placeholders:
+    {current} Number of package that is currently emerged
+    {total} Total number of packages that will be emerged
+    {category} Category of the currently emerged package
+    {pkg} Name of the currently emerged packaged
+    {action} Current emerge action
+
+Format examples:
+     {current} / {total}'
+    Result: "3 / 14"
+    '{current} / {total} = {pkg}'
+    Result: "3 / 14 = py3status"
+    '{category} - {pkg}'
+    Result: "x11-misc - py3status"
+
+@author AnwariasEu
+"""
+
+import re
+
+
+class Py3status:
+    """
+    """
+    cache_timeout = 0
+    format = u'EMRG: {current} / {total}'
+    hide_if_zero = False
+
+    def _emergeRunning(self):
+        """
+        Check if emerge is running.
+        Returns true if at least one instance of emerge is running.
+        """
+        try:
+            self.py3.command_output(['pgrep', 'emerge'])
+            return True
+        except:
+            return False
+
+    def post_config_hook(self):
+        if not self.py3.check_commands('emerge'):
+            raise Exception("emerge not installed")
```

Comment:
Look at `moc.py`. Make constant `STRING_NOT_INSTALLED` with string `not_installed`. The prefix name will be `emerge_status` which is a name of this module unless you want to change it to something else like... `emerge`. :-)

Reply:
Thanks for the explanation! Totally makes sense to keep things consistent.